README.md

```diff
@@ -7,14 +6,75 @@ The website of [Apache APISIX®](https://apisix.apache.org/), a cloud-native mic
 
 If you want to write a blog or fix some blog-related issues, please read [Apache APISIX Blog Contributing Guide](http://apisix.apache.org/docs/general/blog) first. Then create a pull request.
 
-## Usage
+--- 
+
+# Contribution
+Refer these basic steps to contribute. 
+## Contents
+- [Environment-SetUp](#clone)
+- [Start server](#build)
+- [Run](#run)
+- [Reference](#reference)
+
+## Clone
+To run & test the project locally on your machine , you need to clone it locally.
+you can do it by :- 
 
 ```sh
+$ git clone https://github.com/apache/apisix-website.git
 $ cd website
+```
+## Build
+```sh
+$ yarn build
+```
+will build the existing project on your local machine.
 
-$ yarn
+**Note:-**
+- Before using yarn build , make sure you are inside the main project folder.
+if your current location on terminal is not inside the project folder then run, 
+```sh
+$ cd website 
+```
+then build script.
+
+## Run
 
+Now the main fun begins!! we will be using docusaurus for running and delployng our website on local server.
+
+One can run , 
+```sh
 $ yarn start
 ```
```

Comment:
You can add some more description in this part.
Normally this works for en-US environment.
```suggestion
```sh
yarn start
```

But for zh-CN enviroment, we need a similar command

```shell
yarn start -- -- locale zh
```

Reply:
In fact, we dont need the '--'

Below cmd is enough.

```sh
yarn start --locale zh
```

Reply:
Yeah, just checked. 

Run `yarn start --locale zh` in `Desktop/apisix-website` is good.